internal/cmd/plugin/install/generate.go

```diff
@@ -98,7 +102,7 @@ func newGenerateCmd() *cobra.Command {
 	cmd.Flags().Int32Var(
 		&replicas,
 		"replicas",
-		0,
+		1,
```

Comment:
hm, why are we doing this?

Reply:
The default is 1, why not make it the default also in the options?

Reply:
I think that's because we rely on the source manifest, which could come with a different value.